### ISSUE TITLE
Dynamic speed cap

### ISSUE THREAD

**Issue by waifsu** [RemainingToast/2b2tau-issues#4]
Make speed cap change depending on how many players are on, when there are little players on allow faster speed and when higher player count make speed cap lower

**Comment by RemainingToast** [comment]
Good idea, my own coded speed limit plugin is already on the cards so I'll be sure to make it change depending on player count

**Comment by LabtechsOnGitHub** [comment]
i agree, though ive noticed that after restart the anticheat/ speed limit gets more stiff? i might just be dumb but i keep getting the Whoops! Too fast! message when ive been flying at the same speed as before, and before i didnt get the error

**Comment by RhysyPiece360** [comment]
So like 2 players its like 200 BPS 3 players 190 Bps and stuff like that?
A hard limit at like 90 BPS
